src/Datasource/EntityTrait.php

```diff
@@ -1386,16 +1386,6 @@ public function setSource(string $alias): static
         return $this;
     }
 
-    /**
-     * Returns a string representation of this object in a human readable format.
-     *
-     * @return string
-     */
-    public function __toString(): string
```

Comment:
The one benefit of this is that you can drop entities into logs and it 'just works'

Reply:
never needed that 😅

Reply:
I also have always done `json_encode($entity, PRETTY_PRINT)` my entities when I _have_ to log them. I was not aware that they implemented Stringable, nor would I necessarily have expected them to.

Reply:
`print_r($entity, true)` is also your friend

Reply:
I never had to log entities either, nor did I remember they implemented Stringable.

Reply:
Fair enough, we can add a deprecation for this and see how it goes.